config/muxcable.py

```diff
@@ -344,17 +413,44 @@ def state(state, port):
                         click.echo(("{}".format(lport)))
                     rc = False
 
+            transceiver_dict[asic_index] = per_npu_statedb[asic_index].get_all(
+                per_npu_statedb[asic_index].STATE_DB, 'TRANSCEIVER_INFO|{}'.format(port))
+            vendor_value = transceiver_dict[asic_index].get("manufacturer", None)
+            model_value = transceiver_dict[asic_index].get("model", None)
+
+            """ This check is required for checking whether or not this port is connected to a Y cable
+            or not. The check gives a way to differentiate between non Y cable ports and Y cable ports.
+            TODO this should be removed once their is support for multiple vendors on Y cable"""
+
+            if vendor_value != VENDOR_NAME and model_value != VENDOR_MODEL:
```

Comment:
I believe this should be `or`, not `and`.

Reply:
fixed